src/sql/src/parsers/show_parser.rs

```diff
@@ -37,11 +37,27 @@ impl<'a> ParserContext<'a> {
             } else {
                 self.unsupported(self.peek_token_as_string())
             }
+        } else if self.consume_token("FULL") {
+            if self.consume_token("TABLES") {
+                self.parse_show_full_tables()
+            } else {
+                self.unsupported(self.peek_token_as_string())
+            }
         } else {
             self.unsupported(self.peek_token_as_string())
         }
     }
 
+    fn parse_show_full_tables(&mut self) -> Result<Statement> {
+        match self.parser.peek_token().token {
+            Token::EOF | Token::SemiColon => Ok(Statement::ShowTables(ShowTables {
+                kind: ShowKind::Full,
+                database: None,
+            })),
+            _ => self.unsupported(self.peek_token_as_string()),
```

Comment:
suggestion: Could we support `[in | FROM] [DATABASE]`? We can add a method `parse_database()` to parse the database name and reuse it here.
https://github.com/GreptimeTeam/greptimedb/blob/0a692aafb09d706b202071233b3e3aa2fe8450b6/src/sql/src/parsers/show_parser.rs#L75-L93

Reply:
Hi @evenyag , I don't understand your suggestion here. Do you mean support for `SHOW FULL TABLES [in | FROM] [DATBASE]`? That is, database and `in`/`from` keywords are optional?

Reply:
Yes. You can refer to MySQL's document for the syntax (`[in | FROM] [DATABASE]`).

https://dev.mysql.com/doc/refman/8.0/en/show-tables.html
```
SHOW [EXTENDED] [FULL] TABLES
    [{FROM | IN} db_name]
    [LIKE 'pattern' | WHERE expr]
```

According to the syntax, `FULL` should be an option for the `ShowTables` statement, not a show kind.
```rust
pub struct ShowTables {
    pub kind: ShowKind,
    pub full: bool,
    pub database: Option<String>,
}
```

Reply:
Ohh, okayy.
I get it now.
Full should be an option like how extended is an option which is how I should have implemented it instead of making it a `ShowKind`.